.byebug_history

```diff
@@ -1,2 +1,6 @@
 exit
+login(ENV['USERNAME_SPREE'], ENV['PASSWORD_SPREE'])
```

Comment:
Remove that file

Reply:
If I remove it then I'll have to change something elsewhere regarding the login, right ^^?

Reply:
Or, do you mean solely buybug file itself? Yes? why? xd

Reply:
Byebug history file is just for logs and backup for you, its not the file we want in repo